test/jasmine/tests/config_test.js

```diff
@@ -193,77 +198,157 @@ describe('config argument', function() {
 
-            mouseEvent('click', labelX, labelY);
+                mouseEvent('click', labelX, labelY);
 
-            var editBox = document.getElementsByClassName('plugin-editable editable')[0];
-            expect(editBox).toBeDefined();
-            expect(editBox.textContent).toBe(text);
-            expect(editBox.getAttribute('contenteditable')).toBe('true');
+                var editBox = document.getElementsByClassName('plugin-editable editable')[0];
+                expect(editBox).toBeDefined();
+                expect(editBox.textContent).toBe(text);
+                expect(editBox.getAttribute('contenteditable')).toBe('true');
+            };
         }
 
         function checkIfDraggable(elClass) {
-            var el = document.getElementsByClassName(elClass)[0];
+            return function() {
+                var el = document.getElementsByClassName(elClass)[0];
 
-            var elBox = el.getBoundingClientRect(),
-                elX = elBox.left + elBox.width / 2,
-                elY = elBox.top + elBox.height / 2;
+                var elBox = el.getBoundingClientRect(),
+                    elX = elBox.left + elBox.width / 2,
+                    elY = elBox.top + elBox.height / 2;
 
-            mouseEvent('mousedown', elX, elY);
-            mouseEvent('mousemove', elX - 20, elY + 20);
+                mouseEvent('mousedown', elX, elY);
+                mouseEvent('mousemove', elX - 20, elY + 20);
 
-            var movedBox = el.getBoundingClientRect();
+                var movedBox = el.getBoundingClientRect();
 
-            expect(movedBox.left).toBe(elBox.left - 20);
-            expect(movedBox.top).toBe(elBox.top + 20);
+                expect(movedBox.left).toBe(elBox.left - 20);
+                expect(movedBox.top).toBe(elBox.top + 20);
 
-            mouseEvent('mouseup', elX - 20, elY + 20);
+                mouseEvent('mouseup', elX - 20, elY + 20);
+            };
         }
 
-        it('should make titles editable', function() {
-            checkIfEditable('gtitle', 'Click to enter Plot title');
+        it('should let edits override editable', function(done) {
+            var data = [{y: [1, 2, 3]}];
+            var layout = {width: 600, height: 400};
+            Plotly.newPlot(gd, data, layout, {editable: true})
+            .then(function() {
+                expect(gd._context.edits).toEqual({
+                    annotationPosition: true,
+                    annotationTail: true,
+                    annotationText: true,
+                    axisTitleText: true,
+                    colorbarPosition: true,
+                    colorbarTitleText: true,
+                    legendPosition: true,
+                    legendText: true,
+                    shapePosition: true,
+                    titleText: true
+                });
+            })
+            .then(function() {
+                return Plotly.newPlot(gd, data, layout, {
```

Comment:
yes, I guess since we [don't remove `gd._context` on purge](https://github.com/plotly/plotly.js/blob/e3eba8a7989d9dfd4e54fa5c6ca147bfff9e8a5e/src/plots/plots.js#L1280-L1281) I could omit `data` and `layout` here and that would be equivalent, just changing the config. I've just been tending to avoid `Plotly.plot` since its effects on existing plots are a little strange.